### PR TITLE
Fix pt-BR translation of CS1628 error text

### PR DESCRIPTION
**Pull request by pedroreys**
Fix the pt-BR translation after the changes from PR #24585

### CODE REVIEW

**Comment by jcouv** [comment]
@pedroreys It looks like the English source string changed again (it now mentions local functions), causing a conflict.

**Comment by pedroreys** [comment]
@jcouv I just updated the PR to fix the conflict.

**Comment by jcouv** [comment]
windows_debug_unit32_prtest  failed on `InteractiveCommandHandlerTests.TestExecuteInInteractiveWithoutSelection` ([details](https://ci.dot.net/job/dotnet_roslyn/job/dev15.7.x/job/windows_debug_unit32_prtest/999/))

@dotnet/roslyn-infrastructure FYI, I'll re-run

```
MESSAGE:
System.TimeoutException : Failed to clean up listeners in a timely manner.\r\n---- System.OperationCanceledException : The operation was canceled.
```

**Comment by jcouv** [comment]
test windows_debug_unit32_prtest  please

**Comment by jcouv** [comment]
test windows_release_unit64_prtest  please

**Comment by jasonmalinowski** [comment]
FYI to @sharwell for that "failed to clean up listeners", as that's some new protections put in place.

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.pt-BR.xlf [hunk 1]
```diff
@@ -4387,7 +4387,7 @@ Um bloco catch() depois de um bloco catch (System.Exception e) poderá capturar
       </trans-unit>
       <trans-unit id="ERR_AnonDelegateCantUse">
         <source>Cannot use ref, out, or in parameter '{0}' inside an anonymous method, lambda expression, query expression, or local function</source>
-        <target state="needs-review-translation">Não é possível usar parâmetro ref ou out "{0}" dentro de um método anônimo, expressão lambda ou expressão de consulta</target>
+        <target state="translated">Não é possível usar parâmetro ref, out, ou in "{0}" dentro de um método anônimo, expressão lambda, expressão de consulta, ou função local</target>
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.pt-BR.xlf]: Should this remain marked as "needs-review-translation"? I am not sure how will this change flow into our internal LOC database, if one still exists. CC @tmeschter

Comment (pedroreys) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.pt-BR.xlf]: let me know which state that should be and I can make the change.

**Comment by AlekseyTs** [comment]
@tmeschter What is the right way to update translations so that the work won't be lost during integrations, etc. ?

**Comment by tmeschter** [comment]
@pedroreys First off, thank you for the contribution!

Historically, the translation of resources in the dotnet\roslyn repo has been handled by Microsoft's "in-house" localization team. They make regular passes over the .xlf files to update existing translations and add new ones as necessary, and they have their own processes and requirements for handling translations. At this time, we're still figuring out how to best incorporate community contributions into those processes.

What this means for your PR is that even if we merge it in it is likely to soon be replaced with a similar-but-perhaps-not-identical translation provided by our localization team. I don't want you to waste your time or run afoul of our localization team's requirements, so until we figure out the right approach we're going to have to hold off on merging the PR.

**Comment by pedroreys** [comment]
thanks for the response, folks.

**Pull request closed without merging**